vendor/github.com/urfave/cli/flag_string.go

```diff
@@ -76,6 +76,10 @@ func (c *Context) String(name string) string {
 	return lookupString(name, c.flagSet)
 }
 
+func (c *Context) StringMultiple(name string) string {
```

Comment:
Files in vendor folders are dependencies that we fetch from elsewhere, and periodically sync. Therefore the files are just copies, and we cannot change them, because it would be overwritten on the next update. Therefore changes to vendor folders are not allowed.

If a change in a vendor folder is needed, it needs to go through that library, so we need to submit a pull request to that repository instead, and update to it after it has been accepted.

It looks like you are not using this function though, nor any other change that you made in the vendor folder, so you can just revert all those changes to their original state.